serverless/sql-databases/how-to/use-row-level-security.mdx

```diff
@@ -0,0 +1,90 @@
+---
+meta:
+  title: How to use Row Level Security with Serverless SQL Database
+  description: This page explains how to use Row Level Security with Serverless SQL Databases
+content:
+  h1: How to use Row Level Security with Serverless SQL Database
+  paragraph: This page explains how to use Row Level Security with Serverless SQL Databases
+tags: sql-databases serverless database row-level-security postgresql postgrest
+dates:
+  validation: 2024-09-24
+  posted: 2024-09-24
+categories:
+  - serverless
+---
+
+Row-Level Security is a database security mechanism that allows access only to specific rows of a table based on a user's role or permissions.
+
+Row Level Security can be actived with Serverless SQL Databases for a maximum of two different roles, having both read and write permissions. This can be used to restrict access to a subset of users with frameworks or tools such as [PostgREST](https://docs.postgrest.org/en/v12/).
```

Comment:
```suggestion
Row Level Security can be activated with Serverless SQL Databases for a maximum of two different roles, having both read and write permissions. This can be used to restrict access to a subset of users with frameworks or tools such as [PostgREST](https://docs.postgrest.org/en/v12/).
```